src/topology.ts

```diff
@@ -70,49 +71,6 @@ export const getNodesReadyToRun = (dag: DAG, data: SnapshotData) => {
   return nodes
 }
 
-/**
- * Call init on resources.
- */
-export const initResources = async (spec: Spec, resources: string[]) => {
```

Comment:
I guess we are removing the concept of `resources` from a node? What was the reasoning behind it? Is it just to simplify the responsibilities and concerns of the topology runner?

Reply:
It really wasn't necessary. Also, removing it makes the input data shape much simpler. Now it's just an object of nodes with deps and run function.

Reply:
Yes, I definitely thing the newer simplified API looks cleaner, good stuff